test.yml

```diff
@@ -0,0 +1,51 @@
+apiVersion: v1
```

Comment:
Is this file unintentionally left over from local manual testing?

Reply:
Oops :) Fixed!